pyserini/search/__main__.py

```diff
@@ -19,18 +19,16 @@
 from pyserini.search.reranker import ClassifierType, PseudoRelevanceClassifierReranker
 
 parser = argparse.ArgumentParser(description='Create a input schema')
-parser.add_argument('-index', type=str, metavar='path to index',
-                    required=True, help='the path to lucene index')
+parser.add_argument('-index', type=str, metavar='path to index', required=True, help='the path to lucene index')
```

Comment:
Can fit in one line. 120 char per line.

Reply:
While you're at it, I just wrote this up: https://github.com/lintool/guide/blob/master/coding-style.md#describing-arguments-in-command-line-applications

I think to be consistent with our guidelines we should switch to `--dash` options?

Reply:
Oh right, I will fix this.